buildout.cfg

```diff
@@ -1,11 +1,17 @@
 [buildout]
-extends = http://dist.plone.org/release/4.3.6/versions.cfg
 extensions = mr.developer
+
+extends =
+    plone-5.1.x.cfg
+    versions.cfg
```

Comment:
If I understood correctly how it works, the plone versione here gets replaced by sed during CI, so I guess this should be `plone-x.x.x.cfg`. But then how do we buildout locally? What am I missing here?
Moreover, I see that `versions.cfg` is also extended inside `plone-5.1.x.cfg`, I guess it's not necessary to extend it twice.

Reply:
Now this is fixed and there are no duplicates of extends